src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -25,8 +25,10 @@ private IndexUtils(){}
     /**
      * Load a Tribble .idx index from disk, checking for out of date indexes and old versions
      * @return an Index, or null if we're unable to load
+     *
+     * @param errorOnOutOfDateIndex If true, will raise a UserException when an out of date index file is detected.
      */
-    public static Index loadTribbleIndex(final File featureFile) {
+    public static Index loadTribbleIndex(final File featureFile, final boolean errorOnOutOfDateIndex) {
```

Comment:
There seem to be many instances were you call this method with a hardcoded false at the end of it, perhaps these can be avoided by adding an overload of this method with a default value of false?